### PR TITLE
update user-repeat task

### PR DESCRIPTION
**Pull request by yiweny**
I have updated the task from **predict if a user will attend an event if they have attended an event in the last 7 days** -> **predict if a user will attend an event if they have attended an event in the last 14 days**. 

With this change, we will make the val and test set to both have more than 200 rows.

### CODE REVIEW

File: relbench/tasks/event.py [hunk 1]
```diff
@@ -81,8 +81,16 @@ def make_table(self, db: Database, timestamps: "pd.Series[pd.Timestamp]") -> Tab
         event_attendees = db.table_dict["event_attendees"].df
         event_interest = db.table_dict["event_interest"].df
         timestamp_df = pd.DataFrame({"timestamp": timestamps})
+        eval_timestamp_len = len(timestamp_df)
```
Comment (rusty1s) [inline relbench/tasks/event.py:84]: Update the doc-string to reflect the change

File: relbench/tasks/event.py [hunk 2]
```diff
@@ -115,6 +123,9 @@ def make_table(self, db: Database, timestamps: "pd.Series[pd.Timestamp]") -> Tab
             """
         ).df()
 
+        if eval_timestamp_len == 1:
+            df = df[df.timestamp == df.timestamp.max()]
```
Comment (rusty1s) [inline relbench/tasks/event.py:127]: what does this do?

File: relbench/tasks/event.py (hunk 1 above)
Comment (yiweny) [reply relbench/tasks/event.py:84]: Done

File: relbench/tasks/event.py (hunk 2 above)
Comment (yiweny) [reply relbench/tasks/event.py:127]: So we append 2 timestamps to the validation set and test set to construct the training data(because val set and test set only contains one timestamp originally). We then remove them to avoid having unnecessary timestamps

File: examples/gnn_node.py [hunk 3]
```diff
@@ -21,8 +21,8 @@
 from relbench.external.graph import get_node_train_table_input, make_pkey_fkey_graph
 
 parser = argparse.ArgumentParser()
-parser.add_argument("--dataset", type=str, default="rel-event")
-parser.add_argument("--task", type=str, default="user-attendance")
+parser.add_argument("--dataset", type=str, default="rel-f1")
+parser.add_argument("--task", type=str, default="driver-top3")
```
Comment (rusty1s) [inline examples/gnn_node.py:25]: let's keep the defaults?

**Review by rusty1s — APPROVED** [review]
